### DISCUSSION TITLE
Add button slider reversal option

### DISCUSSION THREAD

**Discussion by StellarSea** [Clooos/Bubble-Card#749] [Feature requests]
I'm operating blinds and curtains with a button slider.

However, one drawback is that the open and close direction are the opposite of the home assistant basic card.

So I would like to request a reversal option for the specific user's environment and the flexibility of the button slider.

![1](https://github.com/user-attachments/assets/b5c12b9a-1723-456e-985b-2f5fe387f68c)
![2](https://github.com/user-attachments/assets/3a27a401-d5cd-4f46-8cb9-87164103c403)

**Comment by KristofM** [comment]
I was in the same position and solved it by using the cover card (instead of button) and installing the "cover gradient background indicator" module, which has the "invert direction" and "highlight the closed part" options.

See https://github.com/Clooos/Bubble-Card/discussions/1628
